packages/playwright/src/isomorphic/teleReceiver.ts

```diff
@@ -238,6 +228,28 @@ export type JsonOnExitEvent = {
   params: undefined;
 };
 
+export type JsonRequest = JsonOnTestPausedRequest;
+export type JsonResponse = JsonOnTestPausedResponse;
+
+export type JsonOnTestPausedRequest = {
```

Comment:
Can we keep this as a part of `JsonEvent`, and add optional `id` to every event that's only visible in `TeleEmitter` and `TeleReceiver` classes? This way we keep everything as before, but optionally send a response if there was "id" field in the "event"?